src/plugin/storage/persistplugin.js

```diff
@@ -0,0 +1,131 @@
+goog.module('plugin.storage.PersistPlugin');
+goog.module.declareLegacyNamespace();
+
+const browser = goog.require('goog.labs.userAgent.browser');
+const log = goog.require('goog.log');
+const AlertEventSeverity = goog.require('os.alert.AlertEventSeverity');
+const AbstractPlugin = goog.require('os.plugin.AbstractPlugin');
+const Metrics = goog.require('os.metrics.Metrics');
+const {launchConfirm} = goog.require('os.ui.window');
```

Comment:
This line will break when `os.ui.window` is converted to a module, because `launchConfirm` is in a different file (`confirm.js`). It currently only works because `goog.require` returns the namespace if requiring a `goog.provide` file.

We may want to avoid using the return value of `goog.require` for old code, both for this reason and because it's clear at a glance what remaining deps are not modules.